templates/iot/examples/iot-config.yaml

```diff
@@ -19,3 +19,18 @@ spec:
       endpoint:
         secretNameStrategy:
           secretName: iot-mqtt-adapter-tls
+    http:
+      enabled: true
+      replicas: 3
```

Comment:
I would keep default values here. Just make them present to be easy to change, or move to other adapters